lib/elements/elements.dart

```diff
@@ -505,3 +505,49 @@ class _ElementTextProperty implements Property {
   // TODO:
   Stream get onChanged => null;
 }
+
+class TabController {
+  List<TabElement> tabElements;
+
+  TabElement _selectedTab;
+
+  TabController({this.tabElements}) {
```

Comment:
If I do this, I get the error that default values of optional parameters must be constant.